src/rules/prefer-absolute-version-dependencies.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const areVersRangesValid = require('./../validators/dependency-audit').areVersRangesValid;
+const LintIssue = require('./../LintIssue');
+const lintId = 'prefer-absolute-version-dependencies';
+const nodeName = 'dependencies';
+const message = 'You are using an invalid version range. Please use absolute versions.';
+const ruleType = 'dependencies-version-range';
+
+const lint = function(packageJsonData, lintType) {
+  const rangeSpecifier = '=';
```

Comment:
And of course there are also versions like "1.4.*" or "1.*" where its not sufficient at all to check the first character. Which makes me think that there is not an easy way to check if a given version is absolute or not.